lua/astrocommunity/scrolling/vim-smoothie/init.lua

```diff
@@ -0,0 +1,5 @@
+return {
+  "psliwka/vim-smoothie",
+  event = "User AstroFile",
+  init = function() vim.g.smoothie_remapped_commands = { "<C-D>", "<C-U>", "<PageDown>", "<PageUp>" } end,
+}
```

Comment:
According to vim-smoothie: 
vim-smoothie aims for sane defaults, and should work out-of-the-box for most users. In some cases, however, you might want to customize its behavior, by adjusting one or more of the following variables in your vimrc:

I think we should drop the remap.

Reply:
ok